test/js/bun/glob/stress.test.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, test, describe } from "bun:test";
+import { Glob } from "bun";
+import { tempFixturesDir } from "./util";
+
+const paths = [
+  "test/js/bun/glob/fixtures/file.md",
+  "test/js/bun/glob/fixtures/second/file.md",
+  "test/js/bun/glob/fixtures/second/nested/file.md",
+  "test/js/bun/glob/fixtures/second/nested/directory/file.md",
+  "test/js/bun/glob/fixtures/third/library/b/book.md",
+  "test/js/bun/glob/fixtures/third/library/a/book.md",
+  "test/js/bun/glob/fixtures/first/file.md",
+  "test/js/bun/glob/fixtures/first/nested/file.md",
+  "test/js/bun/glob/fixtures/first/nested/directory/file.md",
+  "test/js/bun/glob/fixtures/first/nested/directory/file.json",
+];
+
+tempFixturesDir();
```

Comment:
This code should not be run in the top-level of the file incase the file is loaded but the test is skipped. Instead, it should be within a `beforeAll` callback:

```suggestion
beforeAll(() => {
  tempFixturesDir();
 });
```